sample_data/germany_by_state_since_1990/germany_by_state_since_1990.md

```diff
@@ -0,0 +1,12 @@
+# Sources
+
+## germany_by_state_since_1990.geojson
+Runfola D, Anderson A, Baier H, Crittenden M, Dowker E, Fuhrig S, et al. (2020) 
+geoBoundaries: A global database of political administrative boundaries. 
+PLoS ONE 15(4): e0231866. https://doi.org/10.1371/journal.pone.0231866. 
+Downloaded from: https://github.com/wmgeolab/geoBoundaries/tree/main/releaseData/gbOpen/DEU/ADM1 on 4 July 2022.
+
+## germany_by_population_2021.csv
+Source: Federal Statistics Office of Germany. Accessed via :https://www.destatis.de/EN/Themes/Society-Environment/Population/Current-Population/Tables/population-by-laender.html. Downloaded 30 June 2022.
```

Comment:
At the bottom of the link, it mentions results are based on the 2011 census whereas at the top it mentions the population as of 2021. Can you investigate further to see which is correct?

Reply:
I believe they've taken their 2011 census data and used it to make an estimate of their 2021 population. Thanks for pointing this out: I'll make an amendment once we've decided on our overall preference for type of data!